server/data/manageUsersApiClient.ts

```diff
@@ -3,31 +3,27 @@ import RestClient from './hmppsRestClient'
 import { User } from '../@types/CvlUserDetails'
 import { TokenStore } from './tokenStore'
 
-export type AuthUserDetails = {
+export type PrisonUserDetails = {
   name: string
   activeCaseLoadId: string
 }
 
-export type AuthUserEmail = {
+export type PrisonUserEmail = {
```

Comment:
renamed for consistency with above following client rename from `hmppsAuthClient` to `manageUsersApiClient`